packages/quantic/force-app/main/default/lwc/quanticDidYouMean/quanticDidYouMean.js

```diff
@@ -38,6 +38,23 @@ export default class QuanticDidYouMean extends LightningElement {
    * @type {string}
    */
   @api engineId;
+  /**
+   * Whether to disable automatically applying corrections for queries that would otherwise return no results.
+   * When `disableAutomaticallyCorrectQuery` is `false`, the component automatically triggers a new query using the suggested term.
+   * When `disableAutomaticallyCorrectQuery` is `true`, the component returns the suggested term without triggering a new query.
+   * @api
+   * @type {boolean}
+   */
+  @api disableAutomaticallyCorrectQuery = false;
```

Comment:
Oof that's hard to understand :(

In this case I would at least suggest shortening "Automatically" to "auto" :D 

And probably `disableAutoQueryCorrection` seems like a better name?

Reply:
yeah here I re-used the name from atomic, I thought it was ok to me, but we can change it sure!